virttest/shared/cfg/base.cfg

```diff
@@ -1082,3 +1082,16 @@ uuid_dimm = ""
 #vm_sgx_epc_memdev_sgx0 = epc0
 #vm_sgx_epc_node_sgx0 = 0
 # Note: # if also enable guest numa node with sgx, please also add numa params
+#
+#
+# 
+# 
+# Both qemu-img create and convert can be operated with extent_size_hint
+#its default value is 1M. The avaliable value set range is [1M, 3.5G]. It
+#can be checked by qemu-img info xxx.If it is set to 0, no extent_size_hint
+#info displayed in qemu-img info xxx
+# Set image_extent_size_hint
+#image_extent_size_hint = value1 value2 value3
```

Comment:
hi @aliang123 , if we set image_extent_size_hint to a string like 'value1 value2 value3', which means this param can be a list, i.e. we can set several values to extent_size_hint, refer to your description below, I believe this is a single value param. so what about:

# Set the image's extent size hint, or specify per-image values
#image_extent_size_hint = 0             # No extent_size_hint displayed in qemu-img info command
#image_extent_size[_image1] = 1G  # To set image1's extent_size_hint to 1G

Then we can also remove 'If it is set to 0, no extent_size_hint info displayed in qemu-img info xxx' from above